app/src/main/res/values/strings.xml

```diff
@@ -196,13 +197,13 @@
     <string name="warning_read_only">Grant KeePass DX write access to your SD card to save changes to your database.</string>
     <string name="warning_unmounted">Mount your SD card to create or load your database.</string>
     <string name="warning_empty_password">Do you really want to use an empty string as your password?</string>
-    <string name="warning_no_encryption_key">Are you sure you do not want to use any encryption key?</string>
+    <string name="warning_no_encryption_key">Are you sure you do not want to encrypt your database?</string>
     <string name="version_label">Version: %1$s</string>
     <string name="build_label">Build: %1$s</string>
     <string name="configure_fingerprint">Fingerprint scanning is supported but not set up.</string>
     <string name="scanning_fingerprint">Scanning for fingerprints</string>
     <string name="encrypted_value_stored">Encrypted password stored</string>
-    <string name="fingerprint_invalid_key">Invalid fingerprint key problem. Restore your password.</string>
+    <string name="fingerprint_invalid_key">Invalid fingerprint file. Restore your password.</string>
```

Comment:
It's not a file, it's an exception from the key in the Keystore of the Fingerprint API.